src/ui/components/RightBar.css

```diff
@@ -1,3 +1,43 @@
+@keyframes background-pan {
+  from {
+    background-position: 0% center;
+  }
+
+  to {
+    background-position: -200% center;
+  }
+}
+
+@keyframes scale {
+  from,
+  to {
+    transform: scale(0);
+  }
+
+  50% {
+    transform: scale(1);
+  }
+}
+
+@keyframes rotate {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(180deg);
+  }
+}
+
+.magic-text {
```

Comment:
This does not seem implemented anywhere, nothing is given the class `magic-text`. Please remove or implement :)

Reply:
You are right. There used to be a greeting text that used this. My mistake